content/en/docs/reference/kube-scheduler/profiles.md

```diff
@@ -0,0 +1,174 @@
+1. **Score**: These plugins provide a score to each node that has passed the
+   filtering phase. The scheduler will then select the node with the highest
+   weighted scores sum.
+1. **Reserve**: This is an informational extension point that notifies plugins
+   when resources have being reserved for a given Pod.
+1. **Permit**: These plugins can prevent or delay the binding of a Pod.
+1. **PreBind**: These plugins perform any work required before a Pod is bound.
+1. **Bind**: The plugins bind a Pod to a Node. Bind plugins are called in order
+   and once one has done the binding, the remaining plugins are skipped. At
+   least one bind plugin is required.
+1. **PostBind**: This is an informational extension point that is called after
+   a Pod has been bound.
+1. **UnReserve**: This is an informational extension point that is called if
+   a Pod is rejected after being reserved and put on hold by a Permit plugin.
+   
+## Scheduling plugins
+
+The following plugins, enabled by default, implement one or more of these
+extension points:
+
+
+- `DefaultTopologySpread`: Favors spreading across nodes for Pods that belong to
+  services, replica sets and stateful sets.
+  Extension points: *PreScore*, *Score*.
+- `ImageLocality`: Favors nodes that already have the container images that the
+  Pod runs.
+  Extension points: *Score*.
+- `TaintToleration`: Implements
+  [taints and tolerations](/docs/concepts/configuration/taint-and-toleration/).
+  Implements extension points: *Filter*, *Prescore*, *Score*.
+- `NodeName`: Checks if a Pod spec node name matches the current node.
+  Extension points: *Filter*.
+- `NodePorts`: Checks if a node has free ports for the requested Pod ports.
+  Extension points: *PreFilter*, *Filter*.
+- `NodePreferAvoidPods`: Scores nodes according to the node annotation
+  `scheduler.alpha.kubernetes.io/preferAvoidPods`.
+  Extension points: *Score*.
+- `NodeAffinity`: Implements
+  [node selectors](/docs/concepts/configuration/assign-pod-node/#nodeselector)
+  and [node affinity](/docs/concepts/configuration/assign-pod-node/#node-affinity).
+  Extension points: *Filter*, *Score*.
+- `PodTopologySpread`: Implements
+  [Pod topology spread](/docs/concepts/workloads/pods/pod-topology-spread-constraints/).
+  Extension points: *PreFilter*, *Filter*, *PreScore*, *Score*.
+- `NodeUnschedulable`: Filters out nodes that have `.spec.unschedulable` set to
+  true.
+  Extension points: *Filter*.
+- `NodeResourcesFit`: Checks if the node has all the resources that the Pod is
+  requesting.
+  Extension points: *PreFilter*, *Filter*.
+- `NodeResourcesBallancedAllocation`: Favors nodes that would obtain a more
+  balanced resource usage if the Pod is scheduled there.
+  Extension points: *Score*.
+- `NodeResourcesLeastAllocated`: Favors nodes that have a low allocation of
+  resources.
+  Extension points: *Score*.
+- `VolumeBinding`: Checks if the node has or if it can bind the requested
+  volumes.
+  Extension points: *Filter*.
```

Comment:
@alculquicondor ,
nit: There is something odd with lines 95-97. There is a single period on the line before `VolumeRestrictions`.

Reply:
I don't understand this.